Documentation/Storage-Configuration/requirements.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Requirements
+---
+
+## Software
+
+Check out the [pre-requisite](../Getting-Started/Prerequisites/prerequisites.md) section to get started.
+
+### CPU Architecture
+
+We only support `amd64 / x86_64` and `arm64` architecture.
+
+### Kubernetes
+
+The minimum Kubernetes API version required is `1.16.x`.
```

Comment:
Make sure to update this to 1.17.x as seen in the Prerequisites page

Reply:
sure